docs-chef-io/content/server/upgrade_server_ha_v2.md

```diff
@@ -161,15 +161,203 @@ Upgrading from Chef Backend 1.x to Chef Backend 2.x requires full cluster downti
 
 8. To continue the upgrades on Chef Infra Server frontend nodes using this back-end cluster, see [Upgrade Front-ends Associated with a Chef Backend Cluster]({{< relref "install_server_ha/#upgrading-chef-infra-server-on-the-frontend-machines" >}}).
 
-## Chef Backend 2.x to 3.x Upgrade
+## Chef Backend Upgrade to 3.x
 
-| Running Version | Upgrade To Version | Upgrade Type |
-|---------|---------|------|
-| 2.x | 2.x | Rolling |
-| 2.1 | 3.0 | Rolling |
-| 2.0 | 3.0 | Downtime |
+### Planning
+
+The versions of Chef Backend, Elasticsearch, and Chef Infra Server running your system determines the strategy, index treatment, and steps required for your upgrade.
+
+| Chef Backend Version | Elasticsearch Version | Release Date |
+|:---------|:---------|:------|:------|
+| 3.0.0 | 6.8.23 | January 2022|
+| 2.3.16 | 5.6.16 | January 2022|
+| 2.2.0 | 5.6.16 | September 2020|
+| 2.1.0 | 5.6.16 | February 2020 |
+| 2.0.30 | 5.4.1 | June 2018 |
+| 2.0.1 | 5.4.1 | October 2017 |
+| 1.4.6 | 2.3.1 | June 2017 |
+| 1.3.2 | --- | June 2017 |
+| 1.2.5 | --- | December 2016 |
+| 1.1.2 | --- | August 2016 |
+| 1.0.2 | --- | 2016 |
+
+See the [Chef Backend Release Notes]({{< relref "/release_notes_backend" >}}) for more information.
+
+#### Identify the Front-end Chef Infra Server Versions
+
+Find the Chef Infra Server version on each of your front-end nodes, either from the `/opt/opscode/version-manifest` file or by using the command:
+
+```ruby
+chef-server-ctl version
+```
+
+#### Identify the Back-end Elasticsearch Versions
+
+Find the version of Elasticsearch that created the indexes. Run the following command on any one of the backend node and note `created_string` from the output:
+
+```bash
+curl -XGET localhost:9200/chef/_settings/index.version.created*?pretty\&human
+```
+
+#### Upgrade Strategy
+
+The Elasticsearch upgrade uses a rolling strategy, but the Elasticsearch version used to create the indexes determines if your upgrade is automatic or requires downtime.
```

Comment:
Chef-Backend also uses rolling strategy, which might be what we want to mention in this case. `The Chef Backend upgrade uses a rolling strategy, but the Elasticsearch version used to create the indexes determines if your upgrade is automatic or requires downtime.`